custom-metrics-stackdriver-adapter/README.md

```diff
@@ -0,0 +1,113 @@
+# Custom Metrics - Stackdriver Adapter
+
+----
+
+Custom Metrics - Stackdriver Adapter is an implementation of [Custom Metrics
+API] based on Stackdriver per-pod metrics. It's purpose is to enable pod
+autoscaling based on Stackdriver custom metrics.
+
+----
+
+## Usage guide
+
+This guide shows how to set up Custom Metrics - Stackdriver Adapter and export
+metrics to Stackdriver in a compatible way. Once this is done, you can use
+them to scale your application, following [HPAv2 design].
```

Comment:
Let's do not link to design. Is there any documentation of this feature?

Reply:
I thought there is no good documentation, but actually found one - linked instead of the design.